README.md

```diff
@@ -63,24 +63,17 @@ SEQREPO_ROOT_DIR=/usr/local/share/seqrepo/latest  # replace if using different p
 
 In [analysis/download_s3_files.ipynb](./analysis/download_s3_files.ipynb), `transcript_mapping.tsv`, `MANE.GRCh38.v1.3.summary.txt`, and `LRG_RefSeqGene_20231114` will be downloaded to `./analysis/data` directory. You must update the environment variables to use the full path.
 
-#### Gene Normalizer Installation
+#### Gene Normalizer DynamoDB Installation
 
-You must set up [VICC Gene Normalizer](https://github.com/cancervariants/gene-normalization/tree/v0.1.39).
+You must set up the [VICC Gene Normalizer](https://github.com/cancervariants/gene-normalization/tree/v0.1.39). The DynamoDB instance was used in this analysis. The PostgreSQL instance is not supported for running notebooks.
 
-The source files used during ETL methods have been uploaded to the public s3 bucket. If you would like to re-run the ETL methods using the files in this analysis, download and extract the following:
+##### AWS Environment Variables for DynamoDB
 
-* Ensembl
-  * [ensembl_110.gff3.zip](https://nch-igm-wagner-lab-public.s3.us-east-2.amazonaws.com/variation-normalizer-manuscript/gene-normalizer/ensembl_110.gff3.zip)
-* NCBI
-  * [ncbi_GRCh38.p14.gff.zip](https://nch-igm-wagner-lab-public.s3.us-east-2.amazonaws.com/variation-normalizer-manuscript/gene-normalizer/ncbi_GRCh38.p14.gff.zip)
-  * [ncbi_history_20231114.tsv.zip](https://nch-igm-wagner-lab-public.s3.us-east-2.amazonaws.com/variation-normalizer-manuscript/gene-normalizer/ncbi_history_20231114.tsv.zip)
-  * [ncbi_info_20231114.tsv.zip](https://nch-igm-wagner-lab-public.s3.us-east-2.amazonaws.com/variation-normalizer-manuscript/gene-normalizer/ncbi_info_20231114.tsv.zip)
-* HGNC
-  * [hgnc_20231114.json.zip](https://nch-igm-wagner-lab-public.s3.us-east-2.amazonaws.com/variation-normalizer-manuscript/gene-normalizer/hgnc_20231114.json.zip)
+If you do not have an AWS account, you can keep `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and `AWS_SESSION_TOKEN` as is. Local DynamoDB instances will allow dummy credentials.
```

Comment:
Should we provide instructions for setting up local dynamodb?

Reply:
@bwalsh For the analysis, I just had them download the ddb instance I created using v0.1.39 gene-normalizer. I don't think it's necessary since the ETL takes hours to run. The gene-normalizer readme has instructions for running the ETL methods.

Reply:
@korikuzma perfect.  thanks.